src/main/java/org/example/FiltersPage.java

```diff
@@ -49,6 +53,10 @@ public void clearMaxPriceField() {
         maxPriceField.clear();
     }
 
+    public WebElement getMaxPriceField() {
```

Comment:
Remove this method. You shouldn't be able to "work" with the `maxPriceField` element directly from a class with tests